docs/guides/12-Vue-integration.md

```diff
@@ -0,0 +1,197 @@
+This guide will lead you through the process of integrating Airship in [Vue](https://vuejs.org/).
+
+## Including Styles
```

Comment:
This works if we have `styles` and `css` loaders, right? Please a note that vue-cli includes them by default but, if not, the user must add them.

Reply:
Done!